src/processor_options.js

```diff
@@ -1,10 +1,14 @@
 export const processor_options = {};
 
 // find Linter instance
-const linter_path = Object.keys(require.cache).find(path => path.endsWith('/eslint/lib/linter/linter.js') || path.endsWith('\\eslint\\lib\\linter\\linter.js'));
-if (!linter_path) {
+const linter_paths = Object.keys(require.cache).filter(path => path.endsWith('/eslint/lib/linter/linter.js') || path.endsWith('\\eslint\\lib\\linter\\linter.js'));
+if (!linter_paths.length) {
 	throw new Error('Could not find ESLint Linter in require cache');
 }
+// there may be more than one instance in case of multiple folders in the workspace
+// try to find the one that's inside the same node_modules directory as this plugin
+// if not found for some reason, assume it's the last one in the array
```

Comment:
Could you elaborate why you pick the last one in the array as fallback? The previous behavior was picking the first, maybe we should keep this as fallback behavior.

Reply:
It's because the extension will be loaded only when needed, i.e. when a Svelte file has to be linted for the first time. This also applies to ESLint itself though, so the chances are high that in the case like I had, where I first opened a JS file from the server project (loading the ESLint instance in the server directory and no Svelte plugin instance) and only afterwards a Svelte file from the frontend project (loading the *second* ESLint instance in the frontend directory and this time also the Svelte plugin instance), the right ESLint that belongs to the project in question will be loaded more recently.

Of course this still won't work if the order happens to be something like JS file in frontend in another directory where eslintrc doesn't specify the Svelte plugin -> JS file in server -> Svelte file. But this is less likely in my opinion. What do you think?